server/pkg/controller/file.go

```diff
@@ -802,21 +803,31 @@ func (c *FileController) cleanupDeletedFile(qItem repo.QueueItem) {
 	ctxLogger.Info("Successfully deleted item")
 }
 
-func (c *FileController) getHotDcSignedUrl(objectKey string) (string, error) {
+func (c *FileController) getHotDcSignedUrl(objectKey string, objType ente.ObjectType) (string, error) {
 	s3Client := c.S3Config.GetHotS3Client()
-	r, _ := s3Client.GetObjectRequest(&s3.GetObjectInput{
+	input := &s3.GetObjectInput{
 		Bucket: c.S3Config.GetHotBucket(),
 		Key:    &objectKey,
-	})
+	}
+	input.ResponseContentDisposition = aws.String("attachment")
+	if objType == ente.FILE {
```

Comment:
As discussed offline, can also set if for Thumbnail, rest looks good.

Reply:
Done